node/src/main.rs

```diff
@@ -750,6 +750,10 @@ async fn main() {
                                      "code" => LogCode::TokioContention);
             if timeout < Duration::from_secs(10) {
                 timeout *= 10;
+            } else if std::env::var_os("GRAPH_ALLOW_UNRESPONSIVE").is_none() {
+                // The node is unresponsive, kill it in hopes it will be restarted.
+                crit!(contention_logger, "Node is unresponsive, killing process");
+                std::process::exit(2)
```

Comment:
Shouldn't this be `1` instead of `2`? Exit code `2` stands for "Misuse of shell builtins". 😉

Reply:
Oh didn't know there is a standard for this, we already use `1` for tokio panics. Maybe `3`?

Reply:
`1` is standard for general errors. According to https://stackoverflow.com/a/1535733,
```
#define EX_UNAVAILABLE  69      /* service unavailable */
```
could be an option. But since many scripts only check 0 and 1, I'd stick to 1.

Reply:
interesting, `1` it is.